File: 0009-palindrome-number/0009-palindrome-number.c
bool isPalindrome(int x) {
    int rem=0,num=x;
    double sum=0;
    while(x>0)
    {
        rem=x%10;
        sum=sum*10+rem;
        x/=10;
    }
    if(num==sum)
    {
        return true;
    }
    
        return false;
    
    
}

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        int reversedNum=0,sum=x;
        if(x<0) sum=x*-1;
        while(x!=0){
            int temp = x%10;
            reversedNum = reversedNum*10+temp;
            x/=10;
        }
        return sum==reversedNum;
    }
}